api/src/main/java/net/starype/quiz/api/game/answer/AnswerParserCleanString.java

```diff
@@ -0,0 +1,11 @@
+package net.starype.quiz.api.game.answer;
+
+public class AnswerParserCleanString implements AnswerParser {
+    @Override
+    public Answer process(Answer answer) {
+        String outputAnswer = answer.getAnswerText()
+                .strip()
+                .replaceAll("( )+", " ");
```

Comment:
If you're looking for a regex that replaces all spaces chain by a single space, it's `\\s`

Reply:
I believe ```\\s``` doesn't work but i replace it with ```\\s+```. Thanks for the idea